.github/workflows/parlament.yml

```diff
@@ -0,0 +1,56 @@
+
+
```

Comment:
We might want to get rid of these two extra empty lines, please let me know what do you think?